tests/Fixtures/tags/for/loop_in_else.test

```diff
@@ -0,0 +1,29 @@
+--TEST--
+"for" tag loop behavior in an else clause
+--TEMPLATE--
+{% for iterator in iterators %}
+{% for item in iterator %}
+{% else %}
+{{ loop.previous is null ? 'OK' : 'KO' }}
+{{ loop.next is null ? 'OK' : 'KO' }}
+{{ loop.first ? 'OK' : 'KO' }}
+{{ loop.last ? 'OK' : 'KO' }}
```

Comment:
Should be false?

Reply:
👍  False seems more logical  to me (it's not "an" iteration, so not the last one)

Reply:
I would say that any usage of the `loop` variable inside the `else` block is non-sense anyway. Getting any of those boolean value would be fine IMO (a PHP error due to an uninitialized property in Twig is not)